litmus-portal/authentication/pkg/utils/errors.go

```diff
@@ -2,34 +2,43 @@ package utils
 
 import "errors"
 
-//AppError defines general error's throughout the system
+// AppError defines general error's throughout the system
 type AppError error
 
 var (
 	ErrInvalidCredentials            AppError = errors.New("invalid_credentials")
 	ErrServerError                   AppError = errors.New("server_error")
 	ErrInvalidRequest                AppError = errors.New("invalid_request")
 	ErrStrictPasswordPolicyViolation AppError = errors.New("password_policy_violation")
-	ErrUnauthorised                  AppError = errors.New("unauthorised")
+	ErrUnauthorized                  AppError = errors.New("unauthorized")
 	ErrUserExists                    AppError = errors.New("user_exists")
+	ErrUserNotFound                  AppError = errors.New("user does not exists")
+	ErrWrongPassword                 AppError = errors.New("password doesn't match")
+	ErrUpdatingAdmin                 AppError = errors.New("cannot remove admin")
+	ErrUserRemoved                   AppError = errors.New("your account has been disabled")
+	ErrUserAlreadyRemoved            AppError = errors.New("user already removed")
```

Comment:
The terminology should remain same, either it's removed or disabled. If the user can be re-enabled then it should be disabled.